File: check_db.js
import mongoose from 'mongoose';
import { TestScore } from './models/TestScore.js';

async function run() {
  await mongoose.connect(process.env.MONGODB_URI || 'mongodb+srv://csrluser:csrlpass123@cluster0.abcde.mongodb.net/csrl_db?retryWrites=true&w=majority', { useNewUrlParser: true, useUnifiedTopology: true });
  const docs = await TestScore.find({}).lean();
  const neet = docs.find(d => Object.keys(d).some(k => k.startsWith('NCT')));
  console.log("NEET DOC keys:", Object.keys(neet).filter(k => k.startsWith('NCT')));
  console.log("NCT01 values:", Object.keys(neet).filter(k => k.startsWith('NCT')).reduce((acc, k) => { acc[k] = neet[k]; return acc; }, {}));
  process.exit(0);
}
run();

File: check_db.py
import pymongo
import os

client = pymongo.MongoClient('mongodb+srv://csrluser:csrllogin2024@csrl-backend.6m649.mongodb.net/csrl_db?retryWrites=true&w=majority&appName=csrl-backend')
db = client['csrl_db']
test_scores = db['testscores']

print("Connected to DB.")
for d in test_scores.find({}):
    keys = list(d.keys())
    neet_keys = [k for k in keys if k.startswith('NCT')]
    if neet_keys:
        print("NEET KEYS FOUND:")
        for nk in neet_keys:
            print(f"{nk}: {d[nk]}")
        break
